using-timescaledb/hypertables.md

```diff
@@ -109,26 +120,31 @@ PostGIS geospatial indexes).  During testing, you might check your
 total chunk sizes via the [chunk relation size](#chunk_relation_size)
 function.
 
-**Space partitions**: The use of additional partitioning is a very
-specialized use case.  **Most users will not need to use it.**
+**Space partitions**: Space partitioning is optional but can make
+sense for certain types of data or when using distributed hypertables.
+**In most cases it is not needed for non-distributed hypertables.**
```

Comment:
Given the text below about "not benefit from space partitioning", I think you can just replace the boldface sentence with "Space partitioning does not offer any performance advantages for hypertables that are not distributed". Using boldface for anything but very important information or for structuring have a tendency to make the text hard to read.